packages/rancher-windows-gmsa/charts/templates/deployment.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name}}
+  namespace: {{.Release.Namespace}}
+  labels:
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name }}

```

Comment:
Why don't these also have `chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}`?

Or alternatively, why don't all your other resources have `app: {{ .Release.Name }}`?

As mentioned in another comment somewhere in this review, I think we should standardize on annotations and toss them into a helper function, so that way you can use that by default and add more if you need specific, component-specific annotations later.

Reply:
If there's a specific reason why we want the labels on this to be different than anything else, feel free to comment and skip changes for this though!